zerver/lib/events.py

```diff
@@ -100,6 +100,9 @@ def user_data(row: Dict[str, Any]) -> Dict[str, Any]:
         )
         if not is_bot:
             result['profile_data'] = profiles_by_user_id.get(row['id'], {})
+        else:
+            if get_user_profile_by_id(row['id']).bot_owner_id is not None:
+                result['bot_owner_id'] = get_user_profile_by_id(row['id']).bot_owner_id
```

Comment:
I think you want to use `row['owner_id']` instead, and also to change `realm_user_dict_fields` to include `owner_id`.

Reply:
Did you mean `row['bot_owner_id']`? Because `'owner_id'` kept giving me `KeyError`.
I tried this with `'bot_owner_id'`, but the test for transfer of ownership failed.
It seems to me that after transfer of ownership of a bot, `bot_owner_id` is only changing value after browser refresh.
I am pushing the code for you to review.

Reply:
Yes, sorry, `bot_owner_id`.  It's an implicit field in the backend generated by the `bot_owner` ForeignKey.